worlds/sm64ex/docs/setup_en.md

```diff
@@ -37,19 +37,31 @@ After it's done, the Build list should have another entry titled with what you n
 NOTE: For some reason first start of the game always crashes the launcher. Just restart it.
 If it still crashes, recheck if you typed the launch options correctly (Described in "Joining a MultiWorld Game")
 
-# Manual Compilation (Linux/Windows)
+### Manual Compilation (Linux/Windows)
 
-Dependencies for Linux: `sdl2 glew cmake python make`.
-Dependencies for Windows: `mingw-w64-x86_64-gcc mingw-w64-x86_64-glew mingw-w64-x86_64-SDL2 git make python3 mingw-w64-x86_64-cmake`
-SM64EX will link `jsoncpp` dynamic if installed. If not, it will compile and link statically.
+*Windows Instructions*
 
-1. Clone `https://github.com/N00byKing/sm64ex` recursively
-2. Enter `sm64ex` and copy your Rom to `baserom.REGION.z64` where `REGION` is either `us` or `jp` respectively.
-3. Compile with `make`. For faster compilation set the parameter `-jn` where `n` is the Number of CPU Cores.
+First, install [MSYS](https://www.msys2.org/) as described on the page. DO NOT INSTALL INTO A FOLDER PATH WITH SPACES.
+
+After launching msys2, and update by entering `pacman -Syuu` in the command prompt. Next, install the relevant dependencies by entering `pacman -S unzip mingw-w64-x86_64-gcc mingw-w64-x86_64-glew mingw-w64-x86_64-SDL2 git make python3 mingw-w64-x86_64-cmake`. SM64EX will link `jsoncpp` dynamic if installed. If not, it will compile and link statically.
+
+After this, obtain the code base by cloning the relevant repository manually via `git clone --recursive https://github.com/N00byKing/sm64ex`. Ready your ROM by copying your legally dumped rom into your sm64ex folder (which is usually located at `msys64/home/user`). The name of the ROM needs to be `baserom.REGION.z64` where `REGION` is either `us` or `jp` respectively.
```

Comment:
"usually located at" remove that note. Rather not assume where users were before cloning the repo

Reply:
Changes have been adressed : )